src/explore-education-statistics-admin/src/pages/release/components/ReleaseStatusForm.tsx

```diff
@@ -87,7 +88,7 @@ interface Props {
   release: Release;
   statusPermissions: ReleaseStatusPermissions;
   onCancel: () => void;
-  onSubmit: (values: ReleaseStatusFormValues) => void;
+  onSubmit: (values: ReleaseStatusFormValues) => Promise<void> | void;
```

Comment:
How come this has changed? I can't figure out why this is necessary.

Reply:
It's not very significant, it's something @ntsim suggested to remove the typescript warning `'await' has no effect on the type of this expression.` within the `useFormSubmit` on L117.